src/CategoricalArrays.jl

```diff
@@ -23,7 +23,7 @@ module CategoricalArrays
     include("value.jl")
 
     include("array.jl")
-    include("nullablearray.jl")
+    include("missingarray.jl")
```

Comment:
"missingarray" isn't a great name, but I couldn't find anything better ("missingable" is ugly). Anyway it's internal.